public/video-ui/src/reducers/publishedVideoReducer.js

```diff
@@ -0,0 +1,13 @@
+export default function video(state = null, action) {
```

Comment:
I think it makes sense because it is something that has been explicitly set to null rather than something that has not been defined. Its also what the other reducers use.